.github/CODEOWNERS

```diff
@@ -447,39 +427,35 @@
 /main_cron.py @nithusha21
 /main_mail.py @aks681
 /main.py @DubeySandeep @nithusha21
-/feconf.py @seanlip @nithusha21
+/feconf.py @BenHenning @nithusha21
 /constants*.py @nithusha21
 /assets/constants.ts @nithusha21
 /core/controllers/incoming_emails*.py @aks681
 /core/controllers/tasks*.py @aks681
 /core/domain/email*.py @aks681
-/core/domain/image_service*.py @seanlip
-/core/domain/prod_validation_jobs_one_off*.py @seanlip
-/core/domain/storage_model_audit_jobs_test.py @seanlip
-/core/jobs*.py @seanlip
-/core/platform/ @seanlip
-# TODO(#10619): Add @bansalnitish as owner after 2020-10-31
+/core/domain/image_service*.py @DubeySandeep
+/core/domain/prod_validation_jobs_one_off*.py @vojtechjelinek
+/core/domain/storage_model_audit_jobs_test.py @vojtechjelinek
```

Comment:
[Out of context of this PR] Not sure why we just have a code owner for the test file in the infrastructure section.

Reply:
Apparently this file does not have a corresponding non-test file...